composer/utils/checkpoint.py

```diff
@@ -566,6 +567,23 @@ def dist_cp_load(
                 planner=load_planner,
             )
         except CheckpointException as e:
+            if re.search(r'Size mismatch.+torch.Size([816]).+rng.+cuda', str(e)) is not None:
```

Comment:
don't think this regex is right. think this is `r'Size mismatch.+torch\.Size\(\[816\]\).+rng\.\d+\.cuda'`